doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,230 @@
+  * ``bm_nrf54l15dk/nrf54l10/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l10/s115_softdevice``
+  * ``bm_nrf54l15dk/nrf54l15/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l15/s115_softdevice``
+
+..
+
+* Fixed:
+
+  * Fixed wrong RRAM size on the nRF54L10 emulation target from 1022 to 1012 KiB.
+
+..
+
+* Removed:
+
+  * The template board variants.
+  * The ``no_softdevice`` variants of the ``bm_nRF54l15dk``.
+
+DFU
+===
+
+* The DFU subsystem is now introduced in the SDK.
+  For more details, see :ref:`bm_dfu`.
+
+Logging
+=======
+
+* Fixed an issue with the minimal log mode.
+
+Drivers
+=======
+
+* Added the following drivers:
+
+  * The :ref:`driver_lpuarte` driver.
+  * A SoftDevice-based flash driver implementing the Zephyr flash interface.
+
+Libraries
+=========
+
+* Added the following libraries:
+
+  * The ``bm_zms`` storage. TODO link when ready
+  * The ``Storage`` library.
+  * A Zephyr queue functionality.
+
+..
+
+* ``ble_adv`` library:
+
+  * Changed the default advertising name to ``nRF_BM_device``.
+  * Corrected the :kconfig:option:`BLE_ADV_DIRECTED_ADVERTISING_HIGH_DUTY` Kconfig option.
+
+..
+
+* ``ble_conn_params library`` library:
+
+  * Added:
+
+    * Support for asymmetric data length.
+    * The updated connection parameter values to the ``BLE_CONN_PARAMS_EVT_UPDATED``` event.
+
+  * Renamed slave latency to peripheral latency according to specification.
+
+  * Fixed:
+
+    * An issue where the ATT MTU could be set to a lower value than the current value.
+    * An issue where the list of PHYs supported by the SoftDevice was not respected by the library when doing PHY updates.
+    * Data length timeout issue and the :c:func:`data_length_get` function returning wrong value.
+    * Incorrectly stated maximum ATT MTU value.
+
+..
+
+* ``bm_buttons`` library:
+
+  * Added the use of nrfx critical section API.
+
+Samples
+=======
+
+* General changes:
+
+  * All samples now use the logging module instead of ``printk``.
+
+  * Updated:
+  
+    * The main loop sleep instructions as the :c:func:`sd_app_evt_wait` function is deprecated.
+    * Updated the main function flow to always end up in a loop with sleep instructions and deferred log processing instead of returning from the main function.
+      This ensures that log messages are printed and that the device goes into sleep mode.
+      Returning from main leaves the device in a state that prevents low power sleep and deferred log messages will be stored indefinitely in an internal buffer.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`ble_pwr_profiling_sample` sample.
+
+..
+
+* :ref:`ble_cgms_sample` sample:
+
+  * Fixed an issue where a PHY update request was responded to twice.
+
+..
+
+* :ref:`ble_hrs_sample` sample:
+
+  * Removed the unused Coded PHY Kconfig fragment.
+
+..
+
+* :ref:`ble_nus_sample` sample:
+
+  * Added support for Low Power UARTE (LPUARTE).
+  * Fixed the pin configuration to use the pins from the ``board_config.h`` board header.
```

Comment:
After #333: 
* Fixed an issue in the :c:func:`ble_nus_data_send` function where passing NULL as the nus instance could cause a segmentation fault.